NinchatSDKSwift/Implementations/Managers/Session Manager/NINChatSessionManagerEventHandlers.swift

```diff
@@ -139,6 +139,10 @@ extension NINChatSessionManagerImpl: NINChatSessionManagerEventHandlers {
     
     func onLogEvent(value: String) {
         /// Nothing is here
+        debugger("** GO SDK output: \(value)")
+        if value.contains(Events.connectionSuperseded.rawValue) {
+            self.onActionSessionEvent?(nil, Events.connectionSuperseded, NinchatError(code: 0, title: value))
```

Comment:
This is a weird situation that I didn't figure it out yet. When I try to resume a session with a `session_id`, I get an error in `onLogEvent` instead of `onSessionEvent` delegate method with the `connection_superseded` keyword in it. This is why I have to handle it like it was a session event.

Reply:
There is no reply event for the `close_session` action. Don't expect one, and don't synthesize one. :)

Reply:
I think the `connection_superseded` error is a connection-level error, which is thus handled by the low-level library silently (unless logging is enabled).

Reply:
OK then. I'll remove it